src/Okta.Idx.Sdk.E2ETests/settings.json

```diff
@@ -0,0 +1,14 @@
+﻿{
+  "okta": {
+    "testing": {
+      "NormalUser": "mary@acme.com",
+      "DeactivatedUser": "mary-deactivated@acme.com",
+      "LockedUser": "mary-locked@acme.com",
+      "SuspendedUser": "mary-suspended@acme.com",
+      "UnassignedUser": "mary-unassigned@mailinator.com",
+      "UserPassword": "Abcd1234",
```

Comment:
I don't think we should expose these secrets. I suggest keeping them local and regenerate new ones.

Reply:
all the sensitive will be kept in secure Environment vars when we move to Travis CI, keeping them in the config file makes it easier to deploy tests on another local environment.

And actually those values are not used, they are leftovers from first version when I used pre-defined users, currently we are using A18N profiles. I will clean up those values.